packages/react/src/providers/FuelProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import type { FuelConfig } from '@fuel-wallet/sdk';
+
+import { Connect } from '../ui/Connect';
+
+import { FuelHooksProvider } from './FuelHooksProvider';
+import { FuelUIProvider, type FuelUIProviderProps } from './FuelUIProvider';
+
+export { useFuel } from './FuelHooksProvider';
+export { useConnectUI } from './FuelUIProvider';
+
+type FuelProviderProps = {
+  ui?: boolean;
+  fuelConfig?: FuelConfig;
+} & FuelUIProviderProps;
+
+export function FuelProvider({
+  theme,
+  children,
+  fuelConfig,
+  ui = true,
```

Comment:
I think this is more a personal take, and I think just a config with default values is better them a skip config. The idea is that most SDKs work with default values, instead of making skip config's. Ex. `enabled` from `useQuery`. By default is true if not provide anything, otherwise, it could be `disabled`. But is just a preference.

Reply:
I think If the UI feature is something that will be used most of the time, then it might make sense to have a skipUi prop. This way, the UI feature is enabled by default and developers have to explicitly opt out

On the other hand, if the UI feature is something that will not be used most of the time, then it might be better to have a ui prop that developers can opt into


but it's not a dealbreaker I'm ok with `ui = true`